apps/client/src/app/core/data/localized-data.service.ts

```diff
@@ -292,6 +227,47 @@ export class LocalizedDataService {
     return array[id];
   }
 
+  private getRowWithExtendedLanguage(key: keyof LazyData, id: number | string): I18nName {
+    let row = this.getRow<I18nName>(this.lazyData.data[key], id);
+    if (row === undefined) {
+      return undefined;
+    }
+
+    this.tryFillExtendedLanguage(row, id, this.guessExtendedLanguageKeys(key));
+    return row;
+  }
+
+  private guessExtendedLanguageKeys(key: keyof LazyData) {
+    return {
+      zhKey: this.guessExtendedLanguageKey('zh', key),
+      koKey: this.guessExtendedLanguageKey('ko', key)
+    };
+  }
+
+  private guessExtendedLanguageKey(language: 'zh' | 'ko', key: keyof LazyData): keyof LazyData {
+    let guessKey = `${language}${key.charAt(0).toUpperCase()}${key.substr(1)}`
```

Comment:
Because guessKey is never reassigned, you should use `const` instead of `let` (this breaks linter in CI).
```suggestion
    const guessKey = `${language}${key.charAt(0).toUpperCase()}${key.substr(1)}`
```